haskell-src-meta/src/Language/Haskell/Meta/Extensions.hs

```diff
@@ -216,7 +220,18 @@ fromExtension e = case e of
   TH.NumDecimals                       -> Nothing
   TH.DisambiguateRecordFields          -> Just Exts.DisambiguateRecordFields
   TH.RecordWildCards                   -> Just Exts.RecordWildCards
+#if __GLASGOW_HASKELL__ >= 904
+  TH.NamedFieldPuns                    -> Just Exts.RecordPuns
+  TH.QualifiedDo                       -> Nothing
+  TH.UnliftedDatatypes -> Nothing
+  TH.LinearTypes -> Nothing
+  TH.LexicalNegation -> Nothing
+  TH.FieldSelectors -> Nothing
+  TH.OverloadedRecordDot -> Nothing
+  TH.OverloadedRecordUpdate -> Nothing
```

Comment:
This is dicey, but none of these appear to have corresponding things in `Exts`.

Reply:
I think it is to be expected that HSE lags behind. This should be reviewed again once there is a new major release of HSE.